packages/live-status-gateway/src/wsHandler.ts

```diff
@@ -34,15 +34,22 @@ export abstract class WebSocketTopicBase {
 		this._logger.error(`Process ${this._name} message not expected '${JSON.stringify(msg)}'`)
 	}
 
-	sendMessage(ws: WebSocket, msg: object): void {
+	sendMessage(recipients: WebSocket | Iterable<WebSocket>, msg: object): void {
 		const msgStr = JSON.stringify(msg)
```

Comment:
Perhaps this could be optimised further, by only doing this JSON.stringify if there are any recipients.

The new implementation of this means that if there are none, it will be stringifying when it wouldnt previously